src/test/java/com/kenshoo/xlsx2csv/XlsxToCsvConverterTest.java

```diff
@@ -28,16 +28,21 @@
     private XlsxToCsvConverter xlsxToCsvConverter = new XlsxToCsvConverter();
 
     @Test
-    public void testTranslateFile() {
+    public void testTranslateFile() throws Exception {
         testFileTranslation("bulk_import_test_1.xlsx", "bulk_import_expected_test_1.csv");
     }
 
-    @Test
-    public void testTranslateEmptyFile() {
+    @Test(expected = IOException.class)
```

Comment:
I prefer using JUnit `ExpectedException` rule if you handle many exceptions
1. it allows compile time check of the test code
2. it supports more assertions and verification (for example you can assert on the exception message)